satpy/composites/__init__.py

```diff
@@ -1014,6 +1014,46 @@ def zero_missing_data(data1, data2):
     return data1.where(~nans, 0)
 
 
+class PrecipCloudsRGB(GenericCompositor):
```

Comment:
Could you move this class to the cloudproducts.py file instead ? I think it makes sense to have it there.

Reply:
Can we also rename the module `cloud_products.py`?

Reply:
to what ?

Reply:
Ah misunderstanding. You typed `cloudproducts.py` so I was suggesting `cloud_products.py`. Nevermind then I see it is already called `cloud_products.py`.

Reply:
ah, sorry for the confusion

Reply:
Ok, thanks guys. Done!
I haven't re-tested it yet however. Will do soon.
@mraspaud Should we look at solving the treatment of nodata as well (the white and transparent in the pictures)? Probably we should, just have no clue yet.

Reply:
yes we should. The question is if the reader needs to be fixed or the composite does.

Reply:
I have tested it now. It works again after my last push this morning.
However, the white and nodata thingy I haven't touched. It did not appear in the testcase I had now. I think the issue is that when there is no precip in the AVHRR-only part it becomes nodata and then transparent. Something like this. I will have a short look and see if I can understand the issue. Otherwise I would like to move for another PR and go on. It doesn't really have an affect on the usage. It just doesn't look as nice as it should

Reply:
![ppsv2014_metopb_precipitation_probability_201903282049_euron1](https://user-images.githubusercontent.com/169069/55225314-a5704c80-5212-11e9-8c45-0bdb03fa5e34.jpg)
![ppsv2014_metopb_precipitation_probability_201903281908_euron1](https://user-images.githubusercontent.com/169069/55225320-a7d2a680-5212-11e9-9493-60eaedebda80.jpg)
![ppsv2014_metopb_precipitation_probability_201903281730_euron1](https://user-images.githubusercontent.com/169069/55225327-a99c6a00-5212-11e9-920d-5d652e9e526c.jpg)

I can't seem to find examples of the kind we saw earlier. Could we merge this now, and work on this later if it appears again?